src/components/external-events/ExternalEventForm.svelte

```diff
@@ -0,0 +1,220 @@
+<svelte:options immutable={true} />
+
+<script lang="ts">
+  import { plugins } from '../../stores/plugins';
+  import type { ExternalEventDB } from '../../types/external-event';
+  import type { Property } from '../../types/property';
+  import { classNames } from '../../utilities/generic';
+  import { formatDate } from '../../utilities/time';
+  import Collapse from '../Collapse.svelte';
+  import Input from '../form/Input.svelte';
+  import DatePicker from '../ui/DatePicker/DatePicker.svelte';
+  import Highlight from '../ui/Highlight.svelte';
+  import Properties from './Properties.svelte';
+
+  export let externalEvent: ExternalEventDB;
+  export let showHeader: boolean = true;
+
+  let editable: boolean = false;
```

Comment:
Will this form ever be editable?

Reply:
Good point - no, these fields are always going to be immutable